tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/api/validation/RightToWorkValidator.java

```diff
@@ -25,11 +29,14 @@
 public class RightToWorkValidator {
 
   private static final String DTO_NAME = "RightToWorkDTO";
+  private static String field = "visaIssued";
```

Comment:
I knew I was missing a keyword :facepalm:.  Constants should also be `final` like the one above.  It will be more futureproof for the name to describe how it is different from another `field` constant, e.g. `FIELD_NAME_VISA_ISSUED` to match the convention of `DTO_NAME` or  `VISA_ISSUED_FIELD` / `VISA_ISSUED` for a shorter specific name.